LoggableBehavior.php

```diff
@@ -1,166 +1,174 @@
-<?php
-
-namespace sammaye\audittrail;
-
-use Yii;
-use yii\base\Behavior;
-use yii\db\ActiveRecord;
-
-class LoggableBehavior extends Behavior
-{
-	private $_oldattributes = array();
-
-	public $allowed = array();
-	public $ignored = array();
-	public $ignoredClasses = array();
-
-	public $dateFormat = 'Y-m-d H:i:s';
-	public $userAttribute = null;
-
-	public $storeTimestamp = false;
-	public $skipNulls = true;
-	
-	public function events()
-	{
-		return [
+<?php
+
+namespace sammaye\audittrail;
+
+use Yii;
+use yii\base\Behavior;
+use yii\db\ActiveRecord;
+
+class LoggableBehavior extends Behavior
+{
+	private $_oldattributes = array();
+
+	public $allowed = array();
+	public $ignored = array();
+	public $ignoredClasses = array();
+
+	public $dateFormat = 'Y-m-d H:i:s';
+	public $userAttribute = null;
+
+	public $storeTimestamp = false;
+	public $skipNulls = true;
+	
+	public $actvie = true;

```

Comment:
Hi, yes I have tested this code in a project I'm working. Yes, it is a typo, because I was too lazy to do it properly, and I edit the Loggable file directly from github page :(... sorry.